daemons/ipa-slapi-plugins/libotp/hotp.c

```diff
@@ -93,17 +95,27 @@ import_key(PK11SlotInfo *slot, CK_MECHANISM_TYPE mech, SECItem *key)
                                      PK11_ATTR_SESSION |
                                      PK11_ATTR_PRIVATE |
                                      PK11_ATTR_SENSITIVE, NULL);
-    if (!ekey)
+    if (!ekey) {
+        syslog(LOG_ERR, "libotp: in FIPS, PK11_TokenKeyGenWithFlags failed: %d",
```

Comment:
Does 389-DS call ``openlog(3)`` to pass process and PID information to syslog?

Reply:
libotp can't - there's no library init call, as per [the base OTP/FIPS PR](https://github.com/freeipa/freeipa/pull/1621#issuecomment-369702955)

However, if we assume Linux, "The use of `openlog`() is optional; it will automatically be called by `syslog`() if necessary" per syslog(3).